modules/byo-vpc-multiple-az.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * assemblies/byo-vpc.adoc
+
+[id="byo-vpc-multiple-az_{context}"]
+= BYO VPC with multiple availability zones required fields
+
+. Create six subnets as follows:
++
+|===
+| Availability zone | Subnet name | CIDR IP Address
+| A | `publicA`  | `10.0.0.0/19`
+| A | `privateA` | `10.0.96.0/19`
+| B | `publicB`  | `10.0.32.0/19`
+| B | `privateB` | `10.0.128.0/19`
+| C | `publicC`  | `10.0.64.0/19`
+| C | `privateC` | `10.0.160.0/19`
+|===
+
+. Create a `RouteTable` that is associated with the three `Public` subnets
+. Create a routing rule on the RouteTable for IP address `0.0.0.0/0` to the Internet gateway that was created.
+. Create three NAT gateways in the subnet as follows:
+.. A NAT gateway on `publicA` called `privateA` with an elastic Internet address (EIP).
+.. A NAT gateway on `publicB` called `privateB` with an EIP.
+.. A NAT gateway on `publicC` called `privateC` with an EIP.
+. Create a `RouteTable` that is associated with the three `Private` subnets as follows:
+.. Create a routing rule on the RouteTable for IP address `0.0.0.0/0` to the NAT gateway called `PrivateA`.
```

Comment:
The NAT gateway create in previous commands are using lower case `publicA` . But the names used upper cases in below commands `PrivateA`